src/hct_mis_api/config/settings.py

```diff
@@ -290,19 +293,19 @@ def extend_list_avoid_repeats(list_to_extend: List, extend_with: List) -> None:
     list_to_extend.extend(filter(lambda x: not list_to_extend.count(x), extend_with))
 
 
-GIT_VERSION = env("GIT_VERSION", default="UNKNOWN")
+GIT_VERSION = env("GIT_VERSION")
 HIJACK_PERMISSION_CHECK = "hct_mis_api.apps.utils.security.can_hijack"
 
-REDIS_INSTANCE = env("REDIS_INSTANCE", default="redis:6379")
+REDIS_INSTANCE = env("REDIS_INSTANCE")
 
-CACHE_ENABLED = env("CACHE_ENABLED", default=True)
+CACHE_ENABLED = env("CACHE_ENABLED")
 
 CACHES: Dict[str, Any]
 if CACHE_ENABLED:
     CACHES = {
         "default": {
             "BACKEND": "django_redis.cache.RedisCache",
-            "LOCATION": env("CACHE_LOCATION", default=f"redis://{REDIS_INSTANCE}/1"),
```

Comment:
We can remove the default value for REDIS_INSTANCE but we shouldn't remove this here. There may be a service where we depend on setting up only the REDIS_INSTANCE and now it will require explicitly setting the CACHE_LOCATION too.